zha/application/platforms/binary_sensor/__init__.py

```diff
@@ -155,13 +155,15 @@ class Occupancy(BinarySensor):
 
     _attribute_name = "occupancy"
     _attr_device_class: BinarySensorDeviceClass = BinarySensorDeviceClass.OCCUPANCY
+    _attr_primary_weight = 2
 
 
 @MULTI_MATCH(cluster_handler_names=CLUSTER_HANDLER_HUE_OCCUPANCY)
 class HueOccupancy(Occupancy):
     """ZHA Hue occupancy."""
 
     _attr_device_class: BinarySensorDeviceClass = BinarySensorDeviceClass.OCCUPANCY
+    _attr_primary_weight = 1
```

Comment:
This is the motion entity for Hue sensors, I'd bump this to not tie with the OnOff/opening one, so: `_attr_primary_weight = 2` or maybe even `3`.
We should also consider changing the device class for this one in another PR.